### PR TITLE
PR #3 - Firmware build version with short git hash suffix 

### PR DESCRIPTION
**Pull request by GeminiServer**
New - Version rule and format.

### CODE REVIEW

File: Translation Editor/make_translation.py [hunk 1]
```diff
@@ -7,6 +7,13 @@
 from datetime import datetime
 import sys
 import fontTables
+
+# To cause no confusion with the original firmware version of TS100,
+# which actually has the latest firmware version 2.18
+# Lets go ahead start with version 2.20.0001 BETA
+# Version String: xx.yy.zzzz TAG --> x:Major y:Minor z:Build TAG: REL | BETA | ALPHA
+BUILD_VERSION = 'V2.20.0001 BETA'
```
Comment (Ralim) [inline Translation Editor/make_translation.py]: Any chance you feel like swapping out the REL/Beta/Alpha + build numbers to instead be major.minor.`git short hash`

To make it clear where it came from ?

Comment (Ralim) [reply Translation Editor/make_translation.py]: @GeminiServer

Comment (GeminiServer) [reply Translation Editor/make_translation.py]: Good idea to use the git short hash instead a build number.

**Comment by Ralim** [comment]
@GeminiServer 

Just to check, did you get this working (I'm away from hardware at the moment :cry: )

**Comment by GeminiServer** [comment]
> @GeminiServer
> 
> Just to check, did you get this working (I'm away from hardware at the moment 😢 )

Every pull request is hardware checked! 😎

image

**Pull request merged**